### ISSUE TITLE
Windows binaries from installer unsigned

### ISSUE THREAD

**Issue by x1rubinho1x** [pyfa-org/Pyfa#2043]
When installing on Windows 10 the user is prompted to accept installation of an unsigned package.  Please sign the pyfa Windows packages so the user doesn't have to go through this confusing step when installing/upgrading pyfa.

**Comment by DarkFenX** [comment]
This is not as easy as it sounds. To do that, I have to:

1) Get certificate. Free ones do not exist for code signing (e.g. popular service for TLS certificates is not fit to sign native code https://letsencrypt.org/docs/faq/#technical). According to my corp-mate:
> Miyamoto: the cheapest code signing certs issued I found were smth. like 180 USD per anno
But normally around 300-500 usd annually.
2) Break our automated build chain. Currently travis/appveyor build executable and pack it up into zip and exe for windows installer. To pyfa sign executables and installers, I will have to build everything locally, or split remote build process into several parts: build executable, download, sign it, upload, build installer, download, sign it, upload to release page. Needless to say, it would be VERY inconvenient. And exposing private key to a remote server is a no-go, because then anyone can sign their code with it, even those which are malicious.

I am not willing to pay for the free tool I develop, while causing inconveniences for myself. Unless something changes - like there appears service which provides free certificates, or travis/appveyor support securing apps with my certificate in a safe way, pyfa is not going to change as well.

**Comment by blitzmann** [comment]
Yeah this is something I look into every now and then. But as @darkfenx says, its pricey, and no one is willing to pay the costs for a free tool. There may be companies out there that provide free ones to open source software, but they dont advertise it and I'm sure it depends on popularity, etc.

A not on the build process tho, I do believe we would be able to integrate it without exposing the private key to the public (ofc the build server will need to have it). All build services have private / encrypted environment t variables that can be used to house the key. But that's the minor point, the main point is that unless we can find a benevolent issuer, this is probably not going to happen =/
